app/src/main/java/com/quran/labs/androidquran/view/FitSystemRelativeLayout.java

```diff
@@ -32,41 +22,4 @@ public FitSystemRelativeLayout(@NonNull Context context, AttributeSet attrs, int
     // needed to fix positioning of the ayah toolbar
     setLayoutDirection(LAYOUT_DIRECTION_LTR);
   }
-
-  @Override
-  protected boolean fitSystemWindows(@NonNull Rect insets) {
```

Comment:
i sadly don't think we can remove this yet (even if it's deprecated) because it'll break things (especially moving to/from full screen, etc). i really need to move the code to the more modern ways of doing things here, but please bring this back in the meanwhile.